plaso/storage/file_interface.py

```diff
@@ -308,6 +308,25 @@ def GetEventDataByIdentifier(self, identifier):
     """
     return self._storage_file.GetEventDataByIdentifier(identifier)
 
+  def GetEventDataStream(self):
```

Comment:
Rename to "GetEventDataStreams" to be consistent with other methods (GetWarnings, GetEvents, etc.). This will affect all subclasses, calls etc.

Reply:
good catch